File: GIT-USERS/Jerga/es-primer/Car.js
class Car {
  constructor(brand = "Default Brand", model = "Default Model", year) {
    this.brand = brand;
    this.model = model;
    this.year = year || 2000;
  }

  displayCarInformations() {
    console.log(this.brand + " " + this.model + " " + this.year);
  }
}

module.exports = Car;

File: GIT-USERS/Jerga/dart-intro-course/med_app/car.dart
class Car {
  String brand;
  String model;
  int year;

  Car(this.brand, this.model, this.year);

  Car.named({
    brand, model, year
  }) {
    this.brand = brand;
    this.model = model;
    this.year = year;
  }

  void honk() {
    print('$brand is honking!');
  }

  int transformYearToMinutes() {
    return year * 365 * 24 * 60 * 60;
  }
}
